### ISSUE TITLE
ビルドが失敗する

### ISSUE THREAD

**Issue by nishimotz** [nvdajp/nvdajp#273]
https://ci.appveyor.com/project/TakuyaNishimoto/nvdajp/builds/33289507

```
<testcase classname="System.Robot.chromeTests" name="checkbox labelled by inner element" time="10.555">
<failure message="Specific speech did not occur before timeout: Google Chrome&#10;See NVDA log for dump of all speech." type="AssertionError"></failure>
</testcase>
```

![image](https://user-images.githubusercontent.com/76300/83727751-8a079980-a680-11ea-8e9f-02e31d320e62.png)

**Comment by nishimotz** [comment]
解決したのでいったんクローズします

https://ci.appveyor.com/project/TakuyaNishimoto/nvdajp/builds/33309395
